packages/ui/src/components/Form/dataFormat/DataFormatEditor.scss

```diff
@@ -0,0 +1,7 @@
+.dataformat-metadata-editor {
+  margin-top: 24px;
+}
+
+.pf-v5-c-card {
+  margin-bottom: 24px;
+}
```

Comment:
mmm, this is colliding with the card component in general, we could either provide a more specific class or try to apply this style to another element :thinking: 

![image](https://github.com/KaotoIO/kaoto-next/assets/16512618/83832835-613e-4384-82d2-4196acabacf0)

Reply:
Thanks @lordrip, this was indeed causing issues - moved to enclosing component